stories/FileMetadata.stories.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import FileMetadata from '../src/components/FileMetadata';
+import { testVersionFile } from '../src/test-helpers';
+
+storiesOf('FileMetadata', module).add('default', () => (
+  <FileMetadata file={testVersionFile} />
```

Comment:
We should rather use a `store`. There is a `renderWithStoreAndRouter` helper in `./utils` (in `stories/`).

Reply:
I didn't realize we had the option of using a `store` in a story, which is the main reason I created the `testVersionFile`. I'll change it to use a `store`.